src/patternfly/components/Menu/menu.scss

```diff
@@ -87,6 +111,65 @@
       top: var(--pf-c-menu--m-flyout__menu--Top);
       left: 100%;
     }
+
+    .pf-c-menu__list-item {
+      position: relative;
+    }
+  }
+
+  &.pf-m-drilldown {
+    width: var(--pf-c-menu--m-drilldown--Width);
+    height: var(--pf-c-menu--m-drilldown--Height);
+    overflow: hidden;
+    transition: transform var(--pf-c-menu--m-drilldown--Transition--transform), height var(--pf-c-menu--m-drilldown--Transition--height);
+
+    &.pf-m-drilled-in {
+      // stylelint-disable selector-max-class
+      // target first list in menu
+      > .pf-c-menu__content > .pf-c-menu__list,
+      > .pf-c-menu__list {
+        transform: translateX(-100%);
+      }
+      // stylelint-enable
+    }
+
+    // Nested menu
+    .pf-c-menu {
+      --pf-c-menu--BoxShadow: none;
+
+      position: absolute;
+      top: calc(var(--pf-c-menu--PaddingTop) * -1);
+      left: 100%;
+      width: 100%;
+      transition: transform var(--pf-c-menu--m-drilldown__menu--Transition--transform), visibility var(--pf-c-menu--m-drilldown__menu--Transition--visibility);
```

Comment:
i think you want to break out transform / visibility etc like we have sone in other components e.g. Nav

Reply:
As in this? 

`--pf-c-menu--m-drilldown--c-menu--Transition: transform var(--pf-global--TransitionDuration), visibility var(--pf-global--TransitionDuration);`

`transition: var(--pf-c-menu--m-drilldown--c-menu--Transition);`

Reply:
I would probably keep the component var you have for the transition-duration, so a user can override that without having to override the entire `transition` prop value. 

```
transition: transform var(--pf-c-menu--m-drilldown__menu--TransitionDuration--transform), visibility var(--pf-c-menu--m-drilldown__menu--TransitionDuration--visibility);
```

That's how we did it in tabs https://github.com/patternfly/patternfly/blob/0d2c0bfb9210fa7c19ddd331fd40073b2f713bea/src/patternfly/components/Tabs/tabs.scss#L435